EllisWeb.Gematria/GematriaOptions.cs

```diff
@@ -0,0 +1,67 @@
+﻿namespace EllisWeb.Gematria
+{
+    /// <summary>
+    /// Options when converting a number into its Gematria representation
+    /// </summary>
+    public class GematriaOptions
+    {
+        /// <summary>
+        /// default value for <see cref="IncludeSeparators"/>
+        /// </summary>
+        public const bool DEFAULT_SHOULD_INCLUDE_SEPARATORS = true;
```

Comment:
Just set the default for this on the relevant line

```
public bool IncludeSeparators { get; } = true;
```

Reply:
and remove the const?
in that case, we are forcing the user to always send all parameters to `GematriaOptions` ctor.